### PR TITLE
More infrastructure

### LINKED ISSUE #6: Refactor MNIST examples
* [x] Fix TODOs
* [x] Split out more flags and clean them up
* [x] Create a MNIST CNN example without TFE
* [x] Refactor MNIST TFE
* [x] Refactor local MNIST GAN Graph notebook

### PR DESCRIPTION
**Pull request by oskopek**
Fixes #6

### CODE REVIEW

File: docker/run_config.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def main():
 
     args = parser.parse_args()
 
-    flags_parser.parse(args.flags_file, args.remainder)
+    flags_parser.parse(args.flags_file, remainder=args.remainder)
```
Comment (jendelel) [inline docker/run_config.py]: Can you please comment on this line? It looks like a hyperparameter to the network.

File: models/base.py [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+from keras import backend as K
+
+import numpy as np
+
 import tensorflow as tf
 import tensorflow.contrib.summary as tf_summary
 
-from datetime import datetime
+
+def set_seeds(seed=42, graph=None):
```
Comment (jendelel) [inline models/base.py]: +1

File: docker/run_config.py (hunk 1 above)
Comment (oskopek) [reply docker/run_config.py]: Done

Comment (oskopek) [reply docker/run_config.py]: Resolved.

**Pull request merged**